### PR TITLE
Update sample project readme and test

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

File: packages/algob/sample-project/README.md [hunk 1]
```diff
@@ -1,29 +1,40 @@
-This is a sample project
+This is a sample project (boilerplate `algob` project with ASA deployment and funding a smart contract)
```
Comment (robert-zaremba) [inline packages/algob/sample-project/README.md:1]:
```suggestion
This is a boilerplate `algob` project with ASA deployment and funding a smart contract
```

File: packages/algob/sample-project/README.md [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
   - If you are using TestNet, use https://bank.testnet.algorand.network/ to fund address.
   - If you are using Private Network, use `goal clerk send`
   (https://developer.algorand.org/docs/reference/cli/goal/clerk/send/) to fund address.
 
   - After this specify your `network configurations`.
 
+* Install `Algo Builder`
+
+  - `yarn add @algo-builder/algob` or
+
```
Comment (robert-zaremba) [inline packages/algob/sample-project/README.md:17]:
```suggestion
```

File: packages/algob/sample-project/test/sample-test.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { getProgram } from '@algo-builder/algob';
-import { Runtime, StoreAccount, types } from '@algo-builder/runtime';
+const { getProgram } =  require('@algo-builder/algob');
+const { Runtime, StoreAccount, types } = require('@algo-builder/runtime');
```
Comment (robert-zaremba) [inline packages/algob/sample-project/test/sample-test.js:2]: why we need to use `require`?

Comment (ratik21) [reply packages/algob/sample-project/test/sample-test.js:2]: We are adding JS files to mocha to run tests [https://github.com/scale-it/algo-builder/blob/master/packages/algob/src/builtin-tasks/test.ts#L15](https://github.com/scale-it/algo-builder/blob/master/packages/algob/src/builtin-tasks/test.ts#L15).

With `import` it was showing: `Error Occurred: Cannot use import statement outside a ..` with `algob test`. So i changed to "import from" to "require"

**Review by man-of-code — APPROVED** [review]

**Pull request merged**